board.py

```diff
@@ -257,66 +253,67 @@ def cardLocationIsValidSpot(self, tile1Location, tile2Location, newCard):
         """
         # This variable is used for condition 1:
         # Check whether both board locations are empty (no tile on neither of them)
-        emptyLocations = False
+        empty_locations = False
         try:
-            emptyLocations = not isinstance(self.board[tile1Location[1]][tile1Location[0]], Tile) and \
-                                not isinstance(self.board[tile2Location[1]][tile2Location[0]], Tile)
+            empty_locations = not isinstance(self.board[tile_1_location[1]][tile_1_location[0]], Tile) and \
+                             not isinstance(self.board[tile_2_location[1]][tile_2_location[0]], Tile)
         # Condition 2: Exception raised when at least one of the locations of the card is out of bounds
         except Exception:
```

Comment:
I think this is where the OutOfBoundsException is raised instead of in the hardcoded conditionals above